database/rrdhost.c

```diff
@@ -1528,6 +1528,16 @@ void rrdhost_cleanup_obsolete_charts(RRDHOST *host) {
     }
 }
 
+void rrdset_check_obsoletion(RRDHOST *host)
+{
+    RRDSET *st;
+    rrdset_foreach_write(st, host) {
+        if (rrdset_last_entry_t(st) < host->trigger_chart_obsoletion_check) {
+            rrdset_flag_set(st, RRDSET_FLAG_OBSOLETE);
```

Comment:
Is it better to use `rrdset_is_obsolete` here?

Reply:
You are of course correct, changed.